node/npm/glide/index.ts

```diff
@@ -175,16 +176,13 @@ function initialize() {
         InfBoundary,
         KeyWeight,
         Boundary,
-        UpdateOptions,
```

Comment:
`UpdateOptions` doesn't exist in Commands.ts (renamed to `UpdateByScore` I believe)